src/features/delegation/DelegateRegistrationForm.tsx

```diff
@@ -153,6 +155,13 @@ export function DelegateRegistrationForm({
       {({ errors }) => (
         <Form className="mt-4 flex flex-1 flex-col justify-between">
           <div className={'space-y-3'}>
+            {delegatedPercent > 0 && (
+              <p className={'text-red-600'}>
+                You are currently delegating{' '}
+                <span className={'font-bold'}>{delegatedPercent}%</span> of your voting power. You
+                can still register as a delegatee, but consider undelegating.
```

Comment:
some people might think that delegation passes thru. like if you delegate to me i can delegate all votes delegated to me to barry. but from my understanding delegation is 1 level deep only. 

so maybe we add "Delegation voting power is unilevel. Not recursive."